publisher/standard/README.md

```diff
@@ -40,17 +40,17 @@ If needed, to close the application, terminate the docker container or use the `
-|  5   | Keep the application main page open or refresh it after awhile and note the UID2 identity state, updated counter, and login information values. | In the background, the UID2 SDK continuously validates whether the advertising token is up-to-date and refreshes it automatically when needed. If the refresh succeeds, the user opts out, or the refresh token expires, the callback function is invoked and the UI elements are updated with the current state of the UID2 identity. For details, see [Workflow States and Transitions](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/sdks/client-side-identity.md#workflow-states-and-transitions) and [Background Token Auto-Refresh](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/sdks/client-side-identity.md#background-token-auto-refresh).                                                                                                    |
-|  6   | To exit the application, click **Log Out**.                                                                                                     | This calls the [UID2 SDK `disconnect()` function](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/sdks/client-side-identity.md#disconnect-void), which clears the UID2 session and the first-party cookie. This call also makes the [UID2 SDK `isLoginRequired()` function](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/sdks/client-side-identity.md#isloginrequired-boolean) return `true`, which presents the user with the login form again.<br/> NOTE: The page displays the **Log Out** button as long as the user identity is valid and refreshable.                                                                                                                                                                                              |
+|  2   | In the text field at the bottom, enter the user email address that you want to use for testing and click **Log In**.                            | This is a call to the `/login` endpoint ([server.js](server.js)). The login initiated on the server side then calls the [POST /token/generate](https://unifiedid.com/docs/endpoints/post-token-generate#decrypted-json-response-format) endpoint and processes the received response.                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
| 2 | In the text field at the bottom, enter the user email address that you want to use for testing and click **Log In**. | This is a call to the `/login` endpoint ([server.js](server.js)). The login initiated on the server side then calls the [POST /token/generate](https://unifiedid.com/docs/endpoints/post-token-generate) endpoint and processes the received response.

I think we don't need the fragment here? -- just link to the doc for the endpoint.